lib/linguist/languages.yml

```diff
@@ -272,6 +272,8 @@ ApacheConf:
 Apex:
   type: programming
   extensions:
+  - ".apxc"
```

Comment:
The primary extension for the language should remain the first in the list, with the following sorted alphabetically. Is `.apxc` the primary extension fro Apex or would that still be `.cls`?

Reply:
Yes, .apxc is the primary extension for Apex class. Example from oficial website : https://trailhead.salesforce.com/en/content/learn/modules/developer_console/developer_console_source_code